config/vufind/SierraRest.ini

```diff
@@ -36,6 +36,14 @@ use_prefixed_ids = false
 ; Default is 2. Set to 0 to disable retries.
 ;http_retries = 2
 
+; Translation prefix to use for messages coming from Sierra. By default a prefix is
+; not used, but a prefix may be used to distinguish the codes from any other
+; translations (or other libraries). To use a simple prefix in the default text
+; domain:
+; translationPrefix = "sierra_"
```

Comment:
The comments are a little confusing here; you have text explaining the first example, but no text explaining the second example... and then the "actual" commented-out setting is identical to the second example. How would you feel about eliminating the two example lines, and changing the last sentence to something like the following?

You can use a simple prefix in the default domain (e.g. "sierra_") or a custom text domain (e.g. "ILSMessages::").

Reply:
It was just copied from Alma.ini, but I'll revise.

Reply:
Should be better now.